### ISSUE TITLE
link syntax

### ISSUE THREAD

**Issue by sformel-usgs** [ESIPFed/bds-primer-best-practices#31]
**What section of the best practices does your suggestion relate to?**
all

**Short description of content that should be revised**
Links should be consistently formatted

**Suggested revision**
Based on web accessibility advice from @lbrensk11 , links should be embedded in plain text description of what the link is. Alternative text should be used with 'external page icon' to make it clear it's an external link.  A footnote or second line can be used for additional description.

**Comment by lbrensk11** [comment]
I made updates to the links on the Context & Understandability page. Apparently we can use "link-external-icon: true" to generate the external page icon in Quarto. We can also set it so open a new window when a link is clicked with "link-external-newwindow: true" I wasn't sure if the best place to add that was in the _quarto.yml or somewhere else to make that change universal across the website?

**Comment by sformel-usgs** [comment]
Thanks for doing that!  Here is what I added to the `_quarto.yml` and seems to be working:

This:
```
format:
  html:
   theme: cosmo
```
updated to:
```
format:
  html:
   theme: cosmo
   link-external-icon: true
   link-external-newwindow: true
   link-external-filter: '^(?:http:|https:)\/\/esipfed\.github\.io'
```
